consultation_analyser/consultations/models.py

```diff
@@ -132,7 +132,37 @@ class Meta(UUIDPrimaryKeyModel.Meta, TimeStampedModel.Meta):
 
 
 class TopicModelMetadata(UUIDPrimaryKeyModel, TimeStampedModel):
-    # TODO -  Some other metadata on the model TBC and link to saved model
+    scatter_plot_data = models.JSONField(default=dict)
+
+    def get_scatter_plot_data_with_detail(self, topic_id_list: list[int]) -> list[dict]:
```

Comment:
As discussed — we'll move this to the pipeline so it's all ready in the database, including summaries, once the LLM has run

Reply:
Now updated and ready for re-review.

Have tested with the scatter plot branch and working as expected (https://github.com/i-dot-ai/consultation-analyser/pull/317).